tests/lib/data/conversation.test.ts

```
@@ -210,19 +207,18 @@ describe("getConversationById", () => {
 
-    const result = await getConversationBySlugAndMailbox("test-slug-mailbox", mailbox.id);
+    const result = await getConversationBySlugAndMailbox("test-slug-mailbox");
 
     expect(result).not.toBeNull();
     expect(result?.id).toBe(conversation.id);
     expect(result?.slug).toBe("test-slug-mailbox");
-    expect(result?.mailboxId).toBe(mailbox.id);
   });
 
   it("returns null if conversation not found", async () => {
     const { mailbox } = await userFactory.createRootUser();
```

ESLint

[error] 'mailbox' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)